exts/omni.example.spawnPrims/tutorial/tutorial.md

```diff
@@ -1,171 +1,144 @@
-    - [Step 4.2: Updating on_click()](#step-42-updating-onclick)
-- [Step 5: Conclusion](#step-5-conclusion)
 
 ## Step 1: Creating an Extension
 
-Omniverse is made up of all kinds of extensions that were created by developers. In this section we will show how to create an extension and show how the code gets reflected back into Omniverse.
-
-<br>
-
-### Step 1.1: Locating the Extension Manager
-
+Omniverse is made up of all kinds of extensions that were created by developers. In this section you will show how to create an extension and demonstrate how the code gets reflected back into Omniverse.
 
-```Step 1.1.1:``` Once Omniverse Code is open select the `Extensions` Manager in the top right.
 
-We can also view other extensions through this tab. If ```Extensions``` tab is not visible make sure it is checked in the Windows tab.
-<br>
+### Step 1.2: Once Omniverse Code is open select the `Extensions` Manager in the top right.
 
-![spawnprim_tutorial1.png](images/spawnprim_tutorial1.png)
+- You can also view other extensions through this tab. If ```Extensions``` tab is not visible make sure it is checked in the Windows tab.
 
-### Step 1.2: Create template extension
+    ![spawnprim_tutorial1.png](images/spawnprim_tutorial1.png)
 
+- At the top left find the plus ➕ icon. Click this icon and select ```New Extension Template Project``` to begin creating an extension.
 
-```Step 1.2.1:``` At the top left find the plus ➕ icon. Click this icon and select ```New Extension Template Project``` to begin creating an extension.
+    | ![spawnprim_tut_png2](images/spawnprim_tutorial2.png) | ![spawnprim_tut_png3](images/spawnprim_tutorial3.png) |
+    |------ |------ |
 
-| ![spawnprim_tut_png2](images/spawnprim_tutorial2.png) | ![spawnprim_tut_png3](images/spawnprim_tutorial3.png) |
-|------ |------ |
 
+From here another window will popup prompting on where to place the folder. For this example you will create it in the default location. 
 
-From here another window will popup prompting on where to place the folder. For this example we will create it in the default location. 
+- Once the save location is selected, hit `Create`
 
-```Step 1.2.2:``` Once the save location is selected, hit `Create`
+    ![spawnprim_tut_png4](images/spawnprim_tutorial4.png)
 
-![spawnprim_tut_png4](images/spawnprim_tutorial4.png)
+It will prompt for a Project Folder Name and Extension Name. 
 
-Then it will prompt for a Project Folder Name and Extension Name. 
+- Enter a Project Folder Name and Extension Name.
 
-```Step 1.2.3:``` Enter a Project Folder Name and Extension Name.
-
-For this example we chose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for our Extension Name.
-
-> 📝 **Note:** Choose the name of an extension differently for publishing. We suggest ```companyName.extdescrip1.descrip2``` 
+- For this example choose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for the Extension Name.
```

Comment:
We need to replace. I "spawnPrims" with "spawn-prims" here and in other places it'll be "spawn_prims". I can take a pass to fix that after Alex's style edits.